app/frontend/src/api/models.ts

```diff
@@ -103,3 +104,21 @@ export interface SpeechConfig {
     isPlaying: boolean;
     setIsPlaying: (isPlaying: boolean) => void;
 }
+
+export type HistoryListApiResponse = {
+    items: {
+        id: string;
+        entra_id: string;
```

Comment:
This is oid, right? Maybe entra_oid would be slightly clearer (versus an Entra group id)

Reply:
This is a typo. I will fix it.

Reply:
Fixed in 0189dfc